src/components/TopBar/Annotations.js

```diff
@@ -48,6 +48,30 @@ export const Annotations = observer(({ store, annotationStore }) => {
     return () => document.removeEventListener('click', handleClick);
   }, []);
 
+  useEffect(() => {
```

Comment:
Are we still counting comments for each like this, I though the annotations now have the `unresolved_comment_count`, and `comment_count` attached when including those in the annotation query?

Reply:
it has unresolved_comment_count and comment_count attached but when we change something like adding new comment or resolving and unresolving a comment I need to refresh this information to change the icon in annotation dropdown, so I need to do this manually updating the informating using the commentStore